frb_example/pure_dart/dart/lib/bridge_definitions.dart

```diff
@@ -682,20 +651,12 @@ class FeedId {
   });
 }
 
-class I32Array2 {
-  final Int32List inner;
-
-  I32Array2(this.inner) : assert(inner.length == 2);
-
-  I32Array2.Unchecked(this.inner);
-
-  int operator [](int index) {
-    return inner[index];
-  }
-
-  void operator []=(int index, int value) {
-    inner[index] = value;
-  }
+class I32Array2 extends NonGrowableListView<int> {
+  I32Array2(Int32List inner)
+      : assert(inner.length == 2),
+        super(inner);
+  I32Array2.unchecked(Int32List inner) : super(inner);
+  Int32List get inner => Int32List.fromList(super.cast());
```

Comment:
Hmm have you tried to use inner like `.inner[0] = 42` and see whether the the original I32Array2 changes? I am not sure about it, because `fromList` is said to copy elements, and `cast` here seems also unnecessary.

Indeed, when calling `.inner`, someone may not expect they get a brand new copy (with O(N) time)

Reply:
I also think the use of `inner` here is a bit out of place, my intention was that the previous implementation had an `inner` field which was used in the `wire2api` section, so I tried to restore that effect in the new implementation as well, but it worked poorly, the new patch has been committed and I'm waiting for the test results to see if it works as expected.

Reply:
@fzyzcjy `CI / Dart (Web) unit tests (pull_request) ` timeout, I don't know if this is accidental or common, and I don't know how to restart this failed CI on github, if you know how to restart this CI to retest this problem, please let me know.

Reply:
Just submit arbitrary code (e.g. add a new line etc), and ci will rerun.

Yes it does sometimes timeout according to @Desdaemon